invenio_rdm_records/services/config.py

```diff
@@ -179,6 +180,7 @@ class RDMRecordCommunitiesConfig(ServiceConfig, ConfiguratorMixin):
     )
 
     schema = RecordCommunitiesSchema
+    communities_schema = CommunitiesSchema
```

Comment:
Minor: `CommunitiesSchema` may also benefit from adding a unique and duplicate id check as in https://github.com/anikachurilova/invenio-rdm-records/blob/22fe2376431cfd84047c33a9b05dd6f5624d3cde/invenio_rdm_records/services/schemas/record_communities.py#L30-L55